src/app/components/ShepContainer/ShepContainer.jsx

```diff
@@ -1,53 +1,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import Store from '@Store';
 import Breadcrumbs from '../Breadcrumbs/Breadcrumbs';
 import LoadingLayer from '../LoadingLayer/LoadingLayer';
 
-import Store from '../../stores/Store';
+class ShepContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: Store.state.isLoading,
```

Comment:
The docs say to use `getState()` so I think `isLoading: Store.getState().isLoading` may be better.

Reply:
Thanks!